php-frontend/src/main/java/org/sonar/plugins/php/api/visitors/Issue.java

```diff
@@ -22,6 +22,10 @@
 import javax.annotation.Nullable;
 import org.sonar.plugins.php.api.tree.Tree;
 
+/**
+ * @deprecated since 3.26 and will be removed by next SonarQube major version update. Use {@link PhpIssue} instead.
+ */
+@Deprecated(since = "3.26", forRemoval = true)
```

Comment:
```suggestion
/**
 * @deprecated since SonarQube 9.9 and will be removed by next major version update. Use {@link PhpIssue} instead.
 */
@Deprecated(since = "9.9", forRemoval = true)
```